### ISSUE TITLE
ThumbsUp reaction causes error in app

### ISSUE THREAD

**Issue by svenseeberg** [RocketChat/Rocket.Chat.Android#2414]
## Describe the bug
When entering a channel or personal chat where a thumbs up reaction was given, the Android app (v3.2.0) does not load the conversation. The error message 
```
com.squareup.moshi.JsonDataException: Expected END_OBJECT but was NAME at path $.messages[0].reactions.:thumbsup:.usernames at $.messages[0].reactions.:thumbsup:.usernames
```
appears. We are currently using Rocket.Chat 1.3.0-dev on our server.

![Screenshot_20190718-090613_Rocket Chat](https://user-images.githubusercontent.com/15197000/61437104-9a196c00-a93c-11e9-8f9e-54d2cb1c37bb.png)

Also, another user reported a similar problem. His app (I don't know the version) said
```
com.squareup.moshi.JsonDataException: Expected a string but was NULL at path $.messages[28].reactions.:thumbsup:.names[0] at $.messages[28].reactions.:thumbsup:.names[0]
```

## To Reproduce
1. Add a thumbs up reaction to any message (via the web site)
2. Open the Android app and switch to the conversation/channel

## Devices and Versions
Your Rocket.Chat.Android version: 3.2.0
Your Rocket.Chat Server version: 1.3.0-dev

Mobile device model and OS version: Oneplus 5T / LinageOS 16.0

**Comment by svenseeberg** [comment]
Could be related to #2275 ?

**Comment by reetp** [comment]
Yes probably a duplicate of 2275. Always have a good search before opening a new issue.

Seem to have been fixed in the is PR:
https://github.com/RocketChat/Rocket.Chat.Android/pull/2280

When testing this sort of issue there is no point on testing on an old version of the app. There may be fixes for your issue.

Test on the latest version - 3.5.0 is in Play Beta.

**Comment by svenseeberg** [comment]
Oh wait, I thought I updated the app. It seems the update (switching from F-Droid to Play Store) did not happen. I expected to be on the current version, but was not (but did not double check). And because I expected to be on the current version, I created a new issue instead of posting on the existing one.

I now removed the App installed via F-Droid and reinstalled from Google Play. Now it works.

I guess that this issue is then about an outdated version on F-Droid.
